kotlinx-coroutines-core/common/src/sync/Mutex.kt

```diff
@@ -93,7 +110,7 @@ public interface Mutex {
  */
 @Suppress("FunctionName")
 public fun Mutex(locked: Boolean = false): Mutex =
-    MutexImpl(locked)
+    MutexImpl(locked) // TODO: locked with owner?
```

Comment:
Should we add a function that specifies the owner?

Reply:
Nope, it exists for ages and there wasn't a single request for that